app.js

```diff
@@ -1,17 +1,3 @@
-process.env.NEW_RELIC_BROWSER_MONITOR_ENABLE = false;
-
-// New Relic Server monitoring support
-var newrelic;
-if ( process.env.NEW_RELIC_ENABLED ) {
```

Comment:
Could you also remove all of the environment variables that aren't used anymore in `env.dist`